pyproject.toml

```diff
@@ -1,183 +1,185 @@
-pyvrp = "pyvrp.cli:main"
-
-
-[tool.black]
-line-length = 79
-
-
-[tool.ruff]
-ignore-init-module-imports = true
-line-length = 79
-select = [
-    "E", "F", "I", "NPY", "PYI", "Q", "RET", "RSE", "RUF", "SLF", "SIM", "TCH"
-]
-
-
-[tool.ruff.isort]
-case-sensitive = true
-known-first-party = ["pyvrp", "tests"]
-
-
-[tool.mypy]
-ignore_missing_imports = true
-
-
-[tool.pytest.ini_options]
-addopts = [
-    "--cov",
-    "--cov-report=xml",
-    "--cov-report=term",
-    "--numprocesses=auto",
-]
-testpaths = "tests"
-
-
-[tool.coverage.run]
-omit = [
-    "build_extensions.py",  # build entrypoint
-    "extract_docstrings.py",  # build script
-    "pyvrp/show_versions.py",  # only prints debug information
-    "pyvrp/cli.py",  # tested in other ways than unit tests
-    ".venv/*",
-    "benchmarks/*",
-    "docs/*",
-    "tests/*",
-]
-
-
-[tool.coverage.report]
-exclude_lines = [
-    "pragma: no cover",
-    "@abstract",
-    "if TYPE_CHECKING:",
-]
-
-
-[tool.cibuildwheel]
-# We do not support old Python versions and somewhat uncommon platforms. For
-# musllinux-based builds we assume users can compile the thing themselves.
-build = "cp310-* cp311-* cp312-* cp313-*"
-skip = "pp* *_ppc64le *_i686 *_s390x *-win32 *-musllinux*"
-build-frontend = "build"
-test-requires = [
-    "pytest>=6.0.0",
-    "pytest-cov>=6.1.1",
-    "pytest-xdist>=3.6.1",
-    "pytest-sugar>=1.0.0",
-]
-test-command = "pytest {package}/tests"
-
-
-[tool.poetry.build]
-generate-setup-file = false
-script = "build_extensions.py"
-
-
-[build-system]
-# We need meson and ninja to build the C++ extensions, and docblock to extract
-# documentation for the extensions.
-requires = ["poetry", "meson", "ninja", "docblock"]
-build-backend = "poetry.core.masonry.api"
+[project]
+name = "pyvrp"
+version = "0.12.0a0"
+description = "A state-of-the-art vehicle routing problem solver."
+authors = [
+    { name = "Niels Wouda", email = "nielswouda@gmail.com" },
+    { name = "Leon Lan", email = "leon.lanyidong@gmail.com" },
+    { name = "Wouter Kool", email = "wouter.kool@ortec.com" },
+]
+license = { file = "LICENSE.md" }
+readme = "README.md"
+keywords = [
+    "vehicle routing problem",
+    "hybrid genetic search",
+    "metaheuristic",    
+]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Development Status :: 5 - Production/Stable",
+    "Topic :: Software Development",
+    "Topic :: Scientific/Engineering",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+requires-python = ">=3.10"
+dependencies = [
+    "numpy>=1.15.2; python_version < '3.12'",
+    "numpy>=1.26.0; python_version >= '3.12'",
+    "matplotlib>=2.2.0",
+    "vrplib>=1.4",
+    "tqdm>=4.64",
+    "tomli>=2.0",
```

Comment:
Question: do we need to specify the patch versions here as well? Looking back it the loss of the patch version is something that happened here: [5c950a1](https://github.com/PyVRP/PyVRP/commit/5c950a147f71f2113ec56fea12fffab229de1ff8)

Reply:
I don't really know. Probably not? But I'm not looking to change our deps here so I'll double check to be sure.